client/components/ProductOverview/ProductOverview.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import mockProductStyles from '../../../MockData/ProductStyles';
+import ImageGallery from './ImageGallery';
+import '../../css/ProductOverviewCSS.css';
+
+const ProductOverview = () => {
+  const currentStyleIndex = 0;
+  const currentStyle = mockProductStyles.results[currentStyleIndex];
+  return (
+    <>
```

Comment:
I'm not sure about this choice. I'm wondering if React.fragment would be better than empty markup?

Reply:
Swapped react.fragments to divs with more descriptive classname